Source/SwiftLintFramework/Rules/ForceUnwrappingRule.swift

```diff
@@ -32,7 +33,7 @@ public struct ForceUnwrappingRule: OptInRule, ConfigurationProviderRule {
     )
 
     public func validateFile(file: File) -> [StyleViolation] {
-        return file.matchPattern("[\\w)]+!", withSyntaxKinds: [.Identifier]).map {
+        return file.matchPattern("([\\w)]+!)|(\"\\)!)", withSyntaxKinds: [.Identifier, .String]).map {
```

Comment:
This seems a little fragile. What about this case: `let url = NSURL(string: "http://www.google.com"  )!` (Spaces before the `)`)